module/zcommon/zfs_prop.c

```diff
@@ -379,8 +391,13 @@ zfs_prop_init(void)
 	zprop_register_index(ZFS_PROP_READONLY, "readonly", 0, PROP_INHERIT,
 	    ZFS_TYPE_FILESYSTEM | ZFS_TYPE_VOLUME, "on | off", "RDONLY",
 	    boolean_table);
+#ifdef __FreeBSD__
+	zprop_register_index(ZFS_PROP_ZONED, "jailed", 0, PROP_INHERIT,
+	    ZFS_TYPE_FILESYSTEM, "on | off", "JAILED", boolean_table);
+#else
```

Comment:
Can this chunk be included latter with the rest of the changes for jails.  I expect there must be additional changes to the common code, at a minimum to the command line tools.

Reply:
there are changes to `/cmd/zfs/zfs_main.c` The rest are in files added to the libzfs for FreeBSD